src/globus_sdk/services/flows/client.py

```diff
@@ -260,6 +265,16 @@ def list_flows(
         the ``flow_starter``, ``flow_administrator`` or ``flow_owner`` roles will be
         returned.
 
+        The role values ``run_manager`` and ``run_monitor`` are also supported but do
+        not follow an order of precedence.
+
+        **Filter Roles Values**
+
+        The valid values for ``filter_roles`` are the same as those for ``filter_role``.
```

Comment:
I wasn't sure how to describe the differences between the `filter_role` and `filter_roles`. I tried to avoid using the term' legacy" and wanted to treat them as separate things. The addition of the `run_managers` and `run_monitors` not being part of the precedence order also made this a bit awkward. I'm open to any suggestions on how to make this documentation better.

Reply:
Thanks! I tried to give what I thought was a pretty good suggestion, so I'm happy with you taking it. :grin: